ibis/backends/dask/execution/util.py

```diff
@@ -146,8 +143,15 @@ def coerce_to_output(
             divisions=(0, out_len - 1),
         )
         return series
+    elif isinstance(result, dd.DataFrame):
+        return result
 
-    return dd.from_pandas(pd_util.coerce_to_output(result, node, index), npartitions=1)
+    try:
```

Comment:
I'm unsure how we would hit this case? And cov says that tests aren't hitting this path.

Reply:
yeah, we weren't hitting that branch. removed now.

Reply:
What is `result` in this case? I'm not sure what non pandas/dask types we'd get here.

Reply:
For reduction ops, `result` can just be of type `int`.